backend/app/adapter/db/url.go

```diff
@@ -39,6 +40,7 @@ WHERE "%s"=$1;`,
 }
 
 // Create inserts a new URL into url table.
+// TODO change to CreateURL
```

Comment:
Let's put TODO on top of the function comment.

Reply:
If I put the TODO at the top then code climate will fail.

Reply:
@oatovar Okey. Then please keep it like this.